hack/istio/install-error-rates-demo.sh

```diff
@@ -81,6 +91,9 @@ echo NAMESPACE_ALPHA=${NAMESPACE_ALPHA}
 echo NAMESPACE_BETA=${NAMESPACE_BETA}
 echo NAMESPACE_GAMMA=${NAMESPACE_GAMMA}
 echo SOURCE=${SOURCE}
+echo DISTRIBUTE_DEMO=${DISTRIBUTE_DEMO}
+echo CLUSTER1_CONTEXT=${CLUSTER1_CONTEXT}
+echo CLUSTER2_CONTEXT=${CLUSTER2_CONTEXT}
```

Comment:
This list is alphabetical, please add these alphabetically (and also fix CLIENT_EXE because it is out of order).

Reply:
Sorted as requested